launch/turtlebot3_navigation.launch.py

```diff
@@ -0,0 +1,59 @@
+# /*******************************************************************************
+# * Copyright 2019 ROBOTIS CO., LTD.
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *     http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *******************************************************************************/
+
+# /* Author: Darby Lim */
+
+import os
+from ament_index_python.packages import get_package_share_directory
+import launch
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+TURTLEBOT3_MODEL = os.environ['TURTLEBOT3_MODEL']
+MAP_PATH = os.environ['HOME']
+
+def generate_launch_description():
+    use_sim_time = launch.substitutions.LaunchConfiguration('use_sim_time', default='false')
+    print('path', os.path.join(
+                      MAP_PATH,
+                      'map.yaml'))
+    map_path = os.path.join(
+                    os.environ['HOME'],
+                    'map.yaml')
+    map_dir = LaunchConfiguration(
+                    'map',
+                    default=map_path)
+    param_file_name = TURTLEBOT3_MODEL + '.yaml'
+    param_dir = LaunchConfiguration(
+        'params',
+        default=os.path.join(
+            get_package_share_directory('person_detection_simulation'),
```

Comment:
What do we still need from PersonDetection? This package should not depend on PersonDetection.

Is it `waffle_pi.yaml` that you're looking for? Can you get it from the `turtlebot3_bringup`, `turtlebot3_navigation2`, or `turtlebot3_node` package?

Reply:
It looks like you're adding a `waffle_pi.yaml` to the `person_detection_simulation` package in https://github.com/aws-robotics/aws-robomaker-sample-application-persondetection/pull/48. Does it contain some information that you can't find in the `waffle_pi.yaml` of the `turtlebot3_bringup`, `turtlebot3_navigation2`, or `turtlebot3_node` packages?

Reply:
Yes, I added following information in waffle_pi.yaml as it is required for initial pose for localization and navigation, which is missing in waffle_pi of the turtlebot3_navigation2, or turtlebot3_node packages:
"initial_pose.x: 3.5
initial_pose.y: 1.0
initial_pose.z: 0.0
initial_pose.yaw: 0.0 "
The above info is application dependent.